### PR TITLE
Easy hardcoded version info

### PR DESCRIPTION
**Pull request by holisticode**
Add new command: print the version number and exit.

closes https://github.com/ava-labs/subnet-cli/issues/32

I have experimented with the goreleaser for a better version string but didn't resolve yet.

NOTE: This version already returns 0.3.3 as version, which requires a new public and tag release after this is merged so that it matches.

### CODE REVIEW

**Review by dasconnor — APPROVED** [review]
If we wanted, we could add `subnet-cli -v` in addition to `subnet-cli version`, but I'm also fine as is

**Comment by aaronbuchwald** [comment]
Agreed that we should use the standard `--version` or `-v` to execute this command.

**Comment by holisticode** [comment]
Updated to use both `--version` and `-v`.
Prints
`subnet-cli version 0.0.3`

Requires manual update on every new release. This version requires new tag 0.0.3 to match.

**Review by dasconnor — APPROVED** [review]
Well, that was easy haha

File: cmd/root.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ import (
 var rootCmd = &cobra.Command{
 	Use:        "subnet-cli",
 	Short:      "subnet-cli CLI",
+	Version:    "0.0.3",
```
Comment (aaronbuchwald) [inline cmd/root.go:17]: Can we move the version string to a variable?

**Review by aaronbuchwald — APPROVED** [review]

**Pull request merged**